src/components/DocumentationTopic/ContentNode.vue

```diff
@@ -30,6 +30,10 @@ export default {
 
 $docs-code-listing-border-width: 1px !default;
 
+@function between-elements-selector($el) {
```

Comment:
I was not sure if to use a mixin or a function. Mixins would create multiple declarations, where as here we merged them into one...

Unfortunately using `space-out-between-siblings` does not work, because the resulting rule was:

`.content-node .element + *, * + *.content-node .element` - which is wrong.... I tried making that function more flexible, but using `&` is just not cutting it, when in combination with `/deep/`.

Reply:
Is this something we even need to create a function/mixin for? It doesn't seem like it's even saving much typing.

Reply:
Just felt a bit easier to read and structure.

Reply:
Even tho it's not a mixin, it may make sense to move this function to a more generic file like _helpers.scss so we can use it outside this component too

Reply:
The problem is we already have something similar. Its just that one has `/deep/` and uses `&` and this does not, because this is nested already in a component. So in a sense, this makes sense to be used only here :/

Reply:
oh, I see... @dobromir-hristov 
Maybe adding this context with a comment in the code would help.